change/@azure-communication-react-a80faf87-e35d-493e-91e9-1e9ecdeb1e43.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
why do we have 2 changelogs for this PR?

Reply:
I made some API changes in second patch and ran `rush changelog` by mistake, looks like extra one got generated